src/components/GraphiQL.js

```diff
@@ -694,6 +710,42 @@ export class GraphiQL extends React.Component {
     editor.setValue(print(parse(editor.getValue())));
   };
 
+  handleDefragQuery = () => {
```

Comment:
Feel like this logic could be under another file

Reply:
Agreed